### DISCUSSION TITLE
如何获取immortalwrt官网下载的固件默认配置的文件

### DISCUSSION THREAD

**Discussion by SlippinDylan** [immortalwrt/immortalwrt#1380] [General]
我按照 GitHub 上的[Development说明](https://github.com/immortalwrt/immortalwrt?tab=readme-ov-file#development)自己编译immortalwrt，因为是首次编译，所以编译完成后的固件很多软件包都没有，与官网的[固件](https://downloads.immortalwrt.org/)差异很多，所以我想请教一下，如何获取到官网固件的默认配置文件，我想应用到我自己的编译中，在官网的基础上进行编译，谢谢。

**Comment by xlighting2017** [comment]
![image](https://github.com/user-attachments/assets/f9b3b5bb-4dc4-458c-b46e-a2abd39eecd7)

**Reply by SlippinDylan** [reply]
是不是需要通过这种方式写进去

`CONFIG_PACKAGE_alsa-utils=y
CONFIG_PACKAGE_autocore=y
CONFIG_PACKAGE_automount=y
CONFIG_PACKAGE_base-files=y
CONFIG_PACKAGE_block-mount=y
CONFIG_PACKAGE_busybox=y
...`
